mm2src/mm2_event_stream/src/event.rs

```diff
@@ -0,0 +1,45 @@
+use serde_json::Value as Json;
+
+// Note `Event` shouldn't be `Clone`able, but rather Arc/Rc wrapped and then shared.
+// This is only for testing.
+#[cfg_attr(test, derive(Clone, Debug, PartialEq))]
+#[derive(Default)]
+/// Multi-purpose/generic event type that can easily be used over the event streaming
+pub struct Event {
+    /// The type of the event (balance, network, swap, etc...).
+    event_type: String,
```

Comment:
There are no dynamic event types, right? If so, this should be static `&str` to make sure we don't utilize something dynamic from runtime.

Reply:
nope, `BALANCE:BTC` is an example of event type. The string is constructed with runtime-computed data.